src/decoder/symphonia.rs

```diff
@@ -345,3 +348,22 @@ impl Iterator for SymphoniaDecoder {
         Some(sample)
     }
 }
+
+#[cfg(feature = "symphonia-mp3")]
+#[cfg(test)]
+mod tests {
```

Comment:
we can make this an integration test right? It only uses public parts of rodio. It would still need the compile time feature but that could test both the minimp3 and symphonia decoder.